Directory.Build.props

```diff
@@ -8,8 +8,8 @@
     <PackageReleaseNotes>https://github.com/G-Research/consuldotnet/blob/master/CHANGELOG.md</PackageReleaseNotes>
     <PackageProjectUrl>https://consuldot.net</PackageProjectUrl>
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
-    <RepositoryType>git</RepositoryType>
-    <RepositoryUrl>https://github.com/G-Research/consuldotnet</RepositoryUrl>
```

Comment:
I was wondering why we don't need to add the reference to the `Microsoft.SourceLink.GitHub` package, but according to the docs (https://github.com/dotnet/sourcelink?tab=readme-ov-file#using-source-link-in-net-projects), it is not needed anymore (starting with .NET 8).

Reply:
yep